help/develop/javascript-development/package-function-types.md

```diff
@@ -0,0 +1,124 @@
+# Function Types
+
+A package can contain various functions, and each function is annotated with a tag that defines what this function does:
+
+* `#app` for [applications](#applications)
+* `#dashboard` for [dashboards](#dashboards)
+* `#panel` for [info panels](#info-panels)
+* `#init` and `#autostart` for [pre-run functions](#pre-run-functions)
+* `#semTypeDetector` for [semantic types detectors](#semantic-type-detectors)
+* `#cellRenderer` for custom [cell renderers](#cell-renderers)
+* `#fileViewer` for [file viewers](#file-viewers)
+* `#fileExporter` for [exporters](#file-exporters)
+* `#packageSettingsEditor` for [settings editors](#package-settings-editors)
+
+You can use the tags to search for certain functions from [Datagrok > Functions] or from within your code:
+
+```js
+const applications = DG.Func.find({ tags: [DG.FUNC_TYPES.APP] });
+```
+
+## Applications
+
+Applications are [functions](../../overview/functions/function.md) tagged with `app`:
+
+```js
+class EnamineStorePackage extends DG.Package {
```

Comment:
This package follows the old class-based format, please consider using another example (https://github.com/datagrok-ai/public/blob/master/packages/BioSignals/src/package.js, https://github.com/datagrok-ai/public/blob/master/packages/ClinicalCase/src/package.ts, etc.)

Reply:
Also, let's provide only a function example itself, without helper functions that do not contribute to understanding.
```js
//name: Test Application
//tags: app
export function app() {
  grok.shell.info('This is Test Application!');
}
```